### PR TITLE
Rebuild with OpenSSL 3 [skip ci]

### PR DESCRIPTION
**Pull request by JeanChristopheMorinPerso**
Rebuild with OpenSSL 3

### CODE REVIEW

**Review by marcoesters — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,18 +26,21 @@ requirements:
...
-    - lz4-c
-    - curl  # [not win]
-    - libcurl  # [not win]
-    - openssl  # [not win]
+    - zlib {{ zlib }}
+    - bzip2 {{ bzip2 }}
+    - zstd {{ zstd }}
+    - lz4-c 1.9.4
+    - curl 7.88.1 # [not win]
+    - libcurl 7.88.1  # [not win]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:40]: Do we really need both curl and libcurl? Because curl has libcurl as a runtime dependency

**Review by skupr-anaconda — APPROVED** [review]
LGTM but curl/libcurl

File: recipe/meta.yaml (hunk 1 above)
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:40]: When I looked at it I had the feeling it was required. I'll double check.

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:40]: See https://github.com/AnacondaRecipes/tiledb-feedstock/blob/master/recipe/build.sh#L18.

**Pull request merged**